Infrastructure/Infrastructure/ProgressTrackers/ApsimXporter.cs

```diff
@@ -39,7 +39,7 @@ public class ApsimXporter : ProgressTracker
         public override int Items => Experiments.Count();
 
         /// <inheritdoc/>
-        public override int Steps => Items * 28;
+        public override int Steps => Items * 29;
```

Comment:
what is 29?

Reply:
A magic number that I haven't spent the time figuring out how to get rid of yet. The progress bar for exporting increments every time a new APSIM model is created and added to the output simulation. 29 is the number of models that are created per simulation, so the number of steps in the progress bar must be a multiple of 29 for the bar to fill.

At some point I plan to find a different way to track the output progress, I've just been prioritising other work.

Reply:
var numModelsToExport = 29;